MyCryptoHQ/MyCrypto
# Address component for checksumming + linking to explorer

Closes #1137 

### Description

Created an `<Address/>` component that takes in a an address or a wallet (just for convenience) and prints a checksummed version of it. You can also pass `network.explorer` and it'll link it to Etherscan or wherever.

### Changes

* Created `<Address/>` component and used it in TransactionStatus, BalanceSidebar, CurrentCustomMessage, and ENS
* (Unrelated) Replaced a few `<a>`s with `<NewTabLink>`s
* (Unrelated) Fixed a bug where wallet create password and confirm password weren't on separate lines

### Steps to Test

1. Unlock your wallet, confirm address appears as checksummed
2. Lookup transaction, confirm address(es) appear as checksummed
3. Lookup ENS, confirm address(es) appear as checksummed
